internal/databases/sqlserver/backup_push_handler.go

```diff
@@ -71,19 +75,24 @@ func HandleBackupPush(dbnames []string, updateLatest bool, compression bool) {
 	tracelog.InfoLogger.Printf("backup finished")
 }
 
-func backupSingleDatabase(ctx context.Context, db *sql.DB, baseUrl string, dbname string, compression bool) error {
+func backupSingleItem(itemName string, ctx context.Context, db *sql.DB,
```

Comment:
Reason: linter complained about duplicate functions: backupSingleDatabase / backupSingleLog

Reply:
Let's disable this linter.